src/Symfony/Component/Validator/Constraints/DefaultPasswordStrengthEstimator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+final class DefaultPasswordStrengthEstimator
+{
+    /**
+     * @var non-empty-array{score: int, minEntropy: int}
+     */
+    private array $criteria = [
```

Comment:
this looks like something that should be a private constant instead of a property.

Reply:
Correct. Done in the last commits